jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java

```diff
@@ -41,67 +47,89 @@ public void before() {
     @Test
     public void closableChipTests() {
         Chip closableChip = closableChips.get(1);
-        closableChip.is().displayed()
-                        .and().removable();
+        Chip greyChip = coloredChips.get(1);
+        closableChip.show();
+        closableChip.is().displayed().and().removable();
         closableChip.close();
         closableChip.is().hidden();
+        greyChip.show();
+        greyChip.is().notRemovable();
     }
 
     @Test
     public void coloredChipTests() {
-        coloredChips.get(1).has().backgroundColor(Colors.GREY_LIGHTEN_2.value());
```

Comment:
The test name is coloredChipTests, but we are working with outlinedChip.  It is not very good. 

The aim of this project is also to be documentation and example, how to work with elements. 
So it should be easy to find correct examples using the test names